src/Tizen.WebView/Tizen.WebView/ContextMenu.cs

```diff
@@ -0,0 +1,554 @@
+        /// Change back.
+        /// </summary>
+        ChangeBack,
+        /// <summary>
+        /// Open media in new window.
+        /// </summary>
+        OpenMediaInNewWindow,
+        /// <summary>
+        /// Copy media link to clipboard.
+        /// </summary>
+        CopyMediaLinkToClipboard,
+        /// <summary>
+        /// Toggle media controls.
+        /// </summary>
+        ToggleMediaControls,
+        /// <summary>
+        /// Toggle media loop.
+        /// </summary>
+        ToggleMediaLoop,
+        /// <summary>
+        /// Enter video fullscreen.
+        /// </summary>
+        EnterVideoFullscreen,
+        /// <summary>
+        /// Media play pause.
+        /// </summary>
+        MediaPlayPause,
+        /// <summary>
+        /// Media mute.
+        /// </summary>
+        MediaMute,
+        /// <summary>
+        /// Dictation alternative.
+        /// </summary>
+        DictationAlternative,
+        /// <summary>
+        /// Select all.
+        /// </summary>
+        SelectAll,
+        /// <summary>
+        /// Select word.
+        /// </summary>
+        SelectWord,
+        /// <summary>
+        /// Text selection mode.
+        /// </summary>
+        TextSelectonMode,
+        /// <summary>
+        /// Clipboard.
+        /// </summary>
+        Clipboard,
+        /// <summary>
+        /// Drag.
+        /// </summary>
+        Drag,
+        /// <summary>
+        /// Translate.
+        /// </summary>
+        Translate,
+        /// <summary>
+        /// Copy link data.
+        /// </summary>
+        CopyLinkData,
+        /// <summary>
+        /// If app want to add customized item, use enum value after BaseApplicationTag.
+        /// </summary>
+        BaseApplicationTag = 10000
+    }
+
+    /// <summary>
+    /// This class provides the properties of Context Menu item.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ContextMenuItem
+    {
+        private IntPtr _handle;
+
+        internal ContextMenuItem(IntPtr handle)
+        {
+            _handle = handle;
+        }
+
+        internal IntPtr GetHandle()
+        {
+            return _handle;
+        }
+
+        /// <summary>
+        /// Gets Tag of the context menu item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public ContextMenuItemTag ItemTag
+        {
+            get
+            {
+                return (ContextMenuItemTag) Interop.ChromiumEwk.ewk_context_menu_item_tag_get(_handle);
+            }
+        }
+    }
+
+    /// <summary>
+    /// Arguments from the ContextMenuItem event.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ContextMenuItemEventArgs : EventArgs
+    {
+        private ContextMenuItem _menuItem;
+
+        internal ContextMenuItemEventArgs(IntPtr handle)
+        {
+            _menuItem = new ContextMenuItem(handle);
+        }
+
+        internal ContextMenuItem GetHandle()
```

Comment:
```suggestion
        public ContextMenuItem GetMenuItem()
```
And also need description for the method.

Reply:
Done